tsconfig.json

```diff
@@ -32,6 +32,7 @@
    { "path": "./packages/common" },
    { "path": "./packages/doctype-caip10-link" },
    { "path": "./packages/doctype-tile" },
+   { "path": "./packages/doctype-tile-handler" },
```

Comment:
Should `doctype-caip10-link-handler` not be added here as well?

Reply:
True.